src/kitchen/package.json

```diff
@@ -27,16 +27,18 @@
   },
   "dependencies": {
     "@guardian/src-button": "^3.9.0",
+    "@guardian/src-helpers": "^3.9.0",
     "@guardian/types": "^6.0.0"
```

Comment:
Types was defined as a dependency, devDependency or peerDependency across the different component `package.json` files. What should it be?

Reply:
peerDeps because Format is in the public API of Editorial Button

Reply:
We should migrate all repos using types to libs, then change to libs here